.github/workflows/release.yml

```diff
@@ -562,8 +579,9 @@ jobs:
           echo "Test version: $BIN_VERSION" >> ${{env.OUTPUT_FILENAME}}
           echo "----------------------------------------------------------------------" >> ${{env.OUTPUT_FILENAME}}
       - name: Compare Metadata
+        timeout-minutes: 10
         run: |
-          CMD="docker run --pull always --network host frequencychain/polkadot-js-tools:v0.52.29 metadata ws://localhost:9946 ws://localhost:9944"
+          CMD="docker run --pull always --net=net-${{env.RELEASE_BRANCH_NAME}} jacogr/polkadot-js-tools:0.55.3 metadata ws://ref-node:9944 ws://test-node:9944"
```

Comment:
If you specify "--pull always" to get the latest version, do you still need to specify 0.55.3?

Reply:
`--pull always` doesn't affect the version that gets pulled. It specifies to always pull the image regardless of whether or not the image is already cached locally.